boilerplate/openshift/golang-osd-operator/operator-sdk-generate.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
```

Comment:
I'd propose putting it under `_lib` as this is essentially acting as a common library function now. It will also allow it to be used by future `golang` `operator-sdk` type conventions.

Reply:
It'll apply to golang operators, so would stay with all the other stuff in the existing convention when we do whatever to split those out.

But we can move it later as needed.

Done.

Reply:
We main need to allign on the purpose of _lib. The way I see it, it contains containing what I would call 'technical pieces' like the ensure.sh script (used by all conventions and fulfilling a generic mission) 
I don't like starting to move functional pieces in _lib, it will make the maintenance more complex as no clue on impacted convention(s) and make the purpose of _lib more obscure. 
At least, we should clarify what we expect in _lib and what we don't expect in it to be alligned.

Reply:
I agree, @bdematte. We've talked about reorganizing the directory structure of the main `boilerplate/` subdirectory to avoid clutter; but IMO this script clearly belongs inside the convention regardless.

Anyone else want to render an opinion? Else I'll move it back.

Reply:
I don't think his question was really answered. What is the criteria for putting a script in the `_lib` directory vs. the convention directories?

Reply:
My take: `_lib` is for things relevant to the boilerplate infrastructure; convention is for things relevant to the consumer itself. There will be some overlap there; IMO convention takes precedence.

IOW: If it ever gets run by the consuming repository *other than* when they're explicitly updating boilerplate, it should go in a convention directory.

In this case, the consumer can run `make opgenerate` any time, not just as part of the "freeze check" we're implementing via #41.

(By this definition, `ensure.sh` is still a bit nebulous. The things it's ensuring right now are in the path of convention-owned `make` targets of general usefulness by the consumer. So we could make a case that it should go in the convention directory. We can do that in another PR if there's consensus on that. @arjunrn wdyt?)

Reply:
@2uasimojo I agree, I only realized later that `_lib` was not the right place for that script.

Reply:
> @2uasimojo I agree, I only realized later that `_lib` was not the right place for that script.

Cool. Moved via #52.

So are we all okay with operator-sdk-generate.sh living in the convention dir as well?

Reply:
Reverted.